content/integrations/slack.md

```diff
@@ -1,13 +1,32 @@
 ---
 title: Datadog-Slack Integration
 integration_title: Slack
+git_integration_title: slack
 kind: integration
-doclevel: basic
+newhlevel: true
 ---
 
+# Overview
 
+Connect Slack to Datadog in order to help your team collaborate by:
 
-* In your Slack account Administration page, go to the Integration tab and choose Datadog.
-* Copy paste the service hook into the field below.
-* Add the channels you want to be able to post to.
-* Tick the checkbox if you want to be notified for every comment, otherwise you will need to use the @slack-channel_name syntax.
+* Sharing graphs with your colleagues in your team's channels.
+* Receiving alerts and notifications from Datadog with in Slack.
+
+
+# Installation
+
+Slack is installed via its [integration tile](https://app.datadoghq.com/account/settings#integrations/slack) in the Datadog application.
+
+# Configuration
+1. In your Slack account  go to the [Applications page](https://slack.com/apps) and search for Datadog.
+1. Click Install, followed by Add Integration.
+1. Copy the Slack Service Hook and paste in the [service hook field for Slack in Datadog](https://app.datadoghq.com/account/settings#integrations/slack).
+1  Add the channels you want to be able to post to.
+1. If you would like to be notified for every comment on a graph tick the check box "Transfer all user comments" by each channel. If left unchecked (default) you will need to use the @slack-channel_name syntax for comments to be posted to slack.
```

Comment:
insert comma: "for every comment on a graph, tick the check box"

s/slack/Slack